cmd/ovnk-mcp-server/main.go

```diff
@@ -0,0 +1,99 @@
+package main
```

Comment:
The current implementation makes it so ovnk-mcp-server will always try to run the k8s mcp. How do you envision the net layers to be implemented? I would imagine the wiring for it to be present as part of the initial PR but I guess we can defer it if that is something that you have a plan for.